connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -485,16 +486,40 @@ public static synchronized void sdkReinitializeMobileConnect() {
         profile.deInitialize();
     }
 
-    public static ConnectivityManager getConnectivityManager() {
-        return connectivityManager;
+    /**
+     * Initialize components common to both Mobile Connect and ConnectID SDK profiles
+     */
+    public static synchronized void initializeCommonComponents() {
```

Comment:
could you not move methods in a file unless you have a good reason to do so.
reading diff is a bit harder.